### ISSUE TITLE
No close button in the top right corner [UX feedback]

### ISSUE THREAD

**Issue by Simon-Laux** [jkelol111/webstore#2]
I expected a close button in the top right corner of the modals, not on the bottom.
![2020-06-10_13-23](https://user-images.githubusercontent.com/18725968/84262181-6f6c6f00-ab1d-11ea-8bba-953479392203.png)
I would move the icon to the left (before the title) and add a close button on the right.

**Comment by jkelol111** [comment]
So something like this?

![Imgur](https://i.imgur.com/wIEAu6p.png)

**Comment by Simon-Laux** [comment]
exactly that looks nice

**Comment by jkelol111** [comment]
Yep I know that, just wanted you to confirm the changes are good enough.
